src/Bicep.Core.IntegrationTests/Extensibility/TestExtensibilityNamespaceProvider.cs

```diff
@@ -6,24 +6,25 @@
 using Bicep.Core.Features;
 using System.Collections.Generic;
 using System.Linq;
+using Bicep.Core.Syntax;
 
 namespace Bicep.Core.IntegrationTests.Extensibility
 {
     public class TestExtensibilityNamespaceProvider : INamespaceProvider
     {
         private readonly INamespaceProvider defaultNamespaceProvider;
 
-        public TestExtensibilityNamespaceProvider(IAzResourceTypeLoader azResourceTypeLoader)
+        public TestExtensibilityNamespaceProvider(IAzResourceTypeLoaderFactory azResourceTypeLoaderFactory)
         {
-            defaultNamespaceProvider = new DefaultNamespaceProvider(azResourceTypeLoader);
+            defaultNamespaceProvider = new DefaultNamespaceProvider(azResourceTypeLoaderFactory);
         }
 
-        public IEnumerable<string> AvailableNamespaces => defaultNamespaceProvider.AvailableNamespaces.Concat(new [] {
+        public IEnumerable<string> AvailableNamespaces => defaultNamespaceProvider.AvailableNamespaces.Concat(new[] {
             StorageNamespaceType.BuiltInName,
             AadNamespaceType.BuiltInName,
         });
 
-        public NamespaceType? TryGetNamespace(string providerName, string aliasName, ResourceScope resourceScope, IFeatureProvider featureProvider)
+        public NamespaceType? TryGetNamespace(string providerName, string aliasName, ResourceScope resourceScope, IFeatureProvider featureProvider, ImportDeclarationSyntax? ids = null)
```

Comment:
```suggestion
        public NamespaceType? TryGetNamespace(string providerName, string aliasName, ResourceScope resourceScope, IFeatureProvider featureProvider, ImportDeclarationSyntax? importDeclaration = null)
```

Reply:
Btw, the reason for the suggestion is that in C# the convention is to not use acronyms when possible. See https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions#using-abbreviations-and-acronyms.

Reply:
Btw, some IDEs like VS will offer completions for variable names based on types, which saves a lot of typing when complying with this rule.